<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyConstraintSubjectActivityTagPreferredRoomForm_template</class>
 <widget class="QDialog" name="ModifyConstraintSubjectActivityTagPreferredRoomForm_template">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string extracomment="This is the title of the dialog to modify a constraint of this type">Modify subject activity tag preferred room</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="subjectsTextLabel">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>Subject</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="buddy">
      <cstring>roomsComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QComboBox" name="subjectsComboBox"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="subjectsTextLabel_2">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>Activity tag</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="buddy">
      <cstring>roomsComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QComboBox" name="activityTagsComboBox"/>
   </item>
   <item row="4" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>53</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="roomsTextLabel">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>Room</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="buddy">
      <cstring>roomsComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QComboBox" name="roomsComboBox"/>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QLabel" name="weightTextLabel">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>Weight percentage (recommended: 0.0%-100.0%)</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QLineEdit" name="weightLineEdit">
     <property name="text">
      <string notr="true">100</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QPushButton" name="okPushButton">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QPushButton" name="cancelPushButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
